Gems/Atom/RPI/Code/Source/RPI.Public/Buffer/Buffer.cpp

```diff
@@ -142,15 +140,16 @@ namespace AZ
                 if (bufferAsset.GetBuffer().size() > 0 && !initWithData)
                 {
                     AZ_PROFILE_SCOPE(RPI, "Stream Upload");
-                    m_streamFence = RHI::Factory::Get().CreateFence();
+                    m_streamFence = aznew RHI::MultiDeviceFence;
                     if (m_streamFence)
                     {
-                        m_streamFence->Init(m_rhiBufferPool->GetDevice(), RHI::FenceState::Reset);
+                        //? TODO: What should be done here?
```

Comment:
Remove this comment or add a GHI ticket number with details on something that will need to be solved later.